x/evm/keeper/params.go

```diff
@@ -57,114 +59,36 @@ func (k Keeper) GetLegacyParams(ctx sdk.Context) types.Params {
 
 // GetExtraEIPs returns the extra EIPs enabled on the chain.
```

Comment:
I suspect that calling all these functions is also negatively affecting performance since `GetParams` is being called multiple times. We need to refactor the logic where this is the case

Reply:
try deleting these functions